scripts/package.ps1

```diff
@@ -7,13 +13,24 @@ Set-ItemProperty -Path $pageFileMemoryKey -Name PagingFiles -Value ""
 
 Update-ExecutionPolicy -Policy Unrestricted
 
-Write-BoxstarterMessage "Removing unused features..."
-Remove-WindowsFeature -Name 'Powershell-ISE'
-Get-WindowsFeature | 
-? { $_.InstallState -eq 'Available' } | 
-Uninstall-WindowsFeature -Remove
+# cmdlet not available in win 7
+if ($OS.Version -ne "6.1.7601") {
```

Comment:
Same principle here. We should do a Get-Command on Remove-WindowsFeature. We would want this on any client sku.